pdr_backend/subgraph/slot.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+
+from enforce_typing import enforce_types
+
+
+@enforce_types
+class Slot:
+    # pylint: disable=too-many-instance-attributes
+    def __init__(
+        self,
+        ID: str,
+        timestamp: int,
+        slot: int,
+        trueval: bool,
```

Comment:
It is decided to use the "truevalue" phrase instead of "trueval". Can you change it ?

Reply:
let's handle this inside this issue #650